itest/lnd_open_channel_test.go

```diff
@@ -357,6 +358,38 @@ func testBasicChannelCreationAndUpdates(ht *lntest.HarnessTest) {
 	runBasicChannelCreationAndUpdates(ht, ht.Alice, ht.Bob)
 }
 
+// testChannelMemoDuringCreation tests the optional Memo argument that can be
+// provided to the openchannel command to serve as a note-to-self for future
+// reference containing some useful information about the channel. We test that
+// the memo is retrieved correctly when querying the created channel. We also
+// test that the 500 character limit is observed correctly.
+func testChannelMemoDuringCreation(ht *lntest.HarnessTest) {
+	// open a channel from Alice to Bob with a valid memo
```

Comment:
nit: Inline comments should also be full sentences with capitalization and punctuation.